rust/lance-encoding/src/statistics.rs

```diff
@@ -165,6 +165,19 @@ impl GetStat for DataBlock {
     }
 }
 
+impl GetStat for FixedSizeListBlock {
+    fn get_stat(&self, stat: Stat) -> Option<Arc<dyn Array>> {
+        match stat {
+            Stat::MaxLength => {
+                let max_len = self.dimension * self.child.expect_single_stat::<UInt64Type>(stat);
+                Some(Arc::new(UInt64Array::from(vec![max_len])))
+            }
+            Stat::DataSize => self.child.get_stat(stat),
+            _ => None,
```

Comment:
What about `NullCount`?

Reply:
removed this code completely as fixed size list statistics is not supported in this PR.